ereolen_support/ereolen_support.form.inc

```diff
@@ -57,7 +57,7 @@ function ereolen_support_form($form, &$form_state) {
     '#type' => 'select',
     '#title' => t('Problem'),
     '#options' => _ereolen_support_create_select_list('ereolen_support_problem', FALSE),
-    '#description' => variable_get('ereolen_support_problem_description')['value'],
+    '#description' => isset(variable_get('ereolen_support_problem_description')['value']) ? variable_get('ereolen_support_problem_description')['value'] : '',
```

Comment:
This could be written as

```php
  '#description' => variable_get('ereolen_support_problem_description')['value'] ?? '',
```

which may be easier to read.